src/features/profile/skills/delete-skills/modal-content.tsx

```diff
@@ -0,0 +1,25 @@
+import Button from "@mui/material/Button";
+import { Box, Typography } from "@mui/material";
+import { style } from "../styles/styles";
+import { Profile } from "core/profiles/types";
+const DeleteSkillModalContent = ({
+  onDeleteSkill,
+  setIsOpen,
+}: DeleteSkillModalContentProps) => {
+  return (
+    <Box sx={style}>
+      <Typography>¿Estás seguro de que quieres borrar la habilidad?</Typography>
+      <Button onClick={() => setIsOpen(false)}>Regresar</Button>
```

Comment:
Yo haria que se inyectara como prop directamente el closeModal mas que el setIsOpen
```suggestion
      <Button onClick={closeModal}>Regresar</Button>
```